### PR TITLE
Cleanup functionality + revamped workflows for new WES + fixes for DB startup errors

### PR DESCRIPTION
**Pull request by davidlougheed**
* Cleanup for 
    * phenopackets:
      * metadata
      * biosamples
      * phenotypic features
    * individuals
    * experiment results
    * instruments
    * resources
* Revamped workflows for Cromwell-based WES
* Service info revisions: URL, email, `bento.serviceKind`
* Functionality to let the entrypoints wait for DB startup

### CODE REVIEW

**Review by brouillette — COMMENTED** [review]
Testing passed on my end

**Review by brouillette — APPROVED** [review]
code looks good

File: chord_metadata_service/experiments/cleanup.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    "clean_instruments",
+]
+
+
+# TODO: Remove this when we have one-to-many ?
+def clean_experiment_results() -> int:
+    results_referenced = set()
+
+    # Collect references to results from experiments
+    results_referenced.update(map(dict_first_val, Experiment.objects.values("experiment_results__id")))
```
Comment (v-rocheleau) [inline chord_metadata_service/experiments/cleanup.py:18]: There is a double '_' in the string "experiment_results__id", not a typo?

**Review by v-rocheleau — COMMENTED** [review]
Tested locally with pr image.

File: chord_metadata_service/experiments/cleanup.py (hunk 1 above)
Comment (davidlougheed) [reply chord_metadata_service/experiments/cleanup.py:18]: not a typo, that's the Django version of `experiment_results.id` - but it does remind me I should test experiments cleanup a bit more thoroughly

**Pull request merged**